### DISCUSSION TITLE
CorsixTH-styled mod for Theme Park (1994)?

### DISCUSSION THREAD

**Discussion by MadeInDutch** [CorsixTH/CorsixTH#2013] [Reception]
I want to know if anyone ever considered a mod for Theme Park like CorsixTH, because Theme Park is still a brilliant game that I think should have a similar update. If not, someone should probably start a project like that IMO. Just an idea.

**Comment by Alberth289346** [comment]
I think you're asking at the wrong spot. To modify the theme park game, the best spot to ask is in a theme park forum.

I am not aware of anyone wanting such a mod, except you. Maybe you should start a project?

**Reply by MadeInDutch** [reply]
Far from a coder myself. And I haven't been successful in finding a Theme Park forum (or an active subreddit).

Just thought it was a good idea.

**Reply by TheCycoONE** [reply]
I know the engine started out the same, but I think it was almost entirely rewritten for Theme Hospital. I'm not sure if there are any tools / documentation on that game's assets or not.

**Reply by lewri** [reply]
A lot of Theme Park's code got borrowed for Theme Hospital, but likely the two differ enough to not be compatible easily with CorsixTH's code.

**Reply by TheCycoONE** [reply]
I was more curious if it was still RNC compression, if the sprite formats were similar etc.

**Comment by MadeInDutch** [comment]
What about a patch for the 2000 European PC port of Theme Aquarium that would allow it to work on modern PCs? I still can't play that despite me owning the disc for it.

**Reply by lewri** [reply]
Never heard of it. Have you tried running in Compatibility mode? 
You could also try dxwnd: https://sourceforge.net/p/dxwnd/home/Home/

**Reply by MadeInDutch** [reply]
Dxwnd doesn't seem to work, compatibility mode never worked, it's a 32-bit game so DOSBox doesn't work, and the only VM that "works" is a Windows 95 VM (even then it doesn't work properly, and I'd much rather have the game working on my modern laptop). The seller sold it as a game that works on Windows XP, but it didn't even work on an XP VM.

https://archive.org/details/theme-aquarium You can download it from here.

**Reply by MadeInDutch** [reply]
https://archive.org/details/aquarium-2000 Or from here.

**Reply by TheCycoONE** [reply]
FWIW there are some dosbox spinoffs that can run Windows 95. If you don't want a VM that might be an option. But there's games from that era I can't get running at all either. It's really the low point of compatibility.

**Reply by MadeInDutch** [reply]
And is it impossible to patch it or something?
